codebuild/spec/buildspec_ktls.yml

```diff
@@ -17,4 +14,10 @@ phases:
   build:
     commands:
       - codebuild-breakpoint
-      - ssh -p 2222 codebuild@localhost "cd s2n-tls; sudo modprobe tls; nix develop .#openssl111 --command bash -c 'source ./nix/shell.sh && clean && configure && build && unit'"
+      - |
```

Comment:
Would love some additional comments here explaining the setup. Or is this already in a readme somewhere? It looks like we're sshing over local host to some other local process? Docker image?

Reply:
If you look at the rest of this file, we're using [qemu](https://www.qemu.org/). I'd say that documentation is outside the scope of this PR-- I'm not modifying the setup, and @dougch did the original setup. I'm just changing some environment variables.